devel/release.md

```diff
@@ -55,17 +65,24 @@ command:
 scripts/update-client.sh
 ```
 
+**NOTE**: If you see a lot of new or modified files under the `kubernetes/test/`
+directory, delete everything except `kubernetes/test/test_api_client.py` and
+`kubernetes/test/test_configuration.py`.
+
 Commit changes (should be only version number changes) to the release branch.
 Name the commit something like "Update version constants for XXX release".
 
-***After you finished the steps above, refer to the section "Hot issues" and
+***After you finished the steps above, refer to the section, "Hot issues", and
 apply the manual fixes.***
 
 ```bash
 git push upstream $RELEASE_BRANCH
 ```
 
-## Hot issues
+## 4. Hot issues
+
+You can use the `scripts/apply-hotfixes.sh` script to apply the fixes below in
```

Comment:
please add a note that the manual fix steps below are deprecated and exist only for documentation purpose.

Reply:
Fixed.